pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -29,6 +29,13 @@ class Meta:
         )
         model = models.CollectionVersion
 
+    def update(self, obj, validated_data):
+        """Update Collection."""
+        collection_data = validated_data.pop("collection")
+        collection = models.Collection.objects.filter(pk=obj.collection.pk)
+        collection.update(**collection_data)
```

Comment:
I think we only want to allow the single field by name. I'm concerned about content becoming "mutable" when we just want that 1 field to be mutable.